src/app/config/projectProperties.json

```diff
@@ -129,6 +129,25 @@
                 "Excel"
             ]
         },
+        "excel-functions-shared": {
+            "displayname": "Excel Shared Runtime Custom Functions Add-in project",
```

Comment:
I don't think there's a name yet. I will keep this PR on hold until we get the name confirmation and then change when we have it

Reply:
@alison-mk can help choose a name. If it were me, I'd like to avoid so many stacked nouns. I'd prefer breaking it up with a prepositional phrase. "Excel Custom Functions with a Shared Runtime project".

Reply:
"Excel Custom Functions using Shared Runtime"  ?

Reply:
We can also change the normal Excel CFs name. Please let me know if we need to do that.

Reply:
I like Ricky and Adam's suggestions. Could we also place Shared Runtime before JS-only in the list, to subtly indicate that it's the preferred method? @igor-ribeiiro Yes, I would also like to change the existing CF template name. I recommend: 

- Excel Custom Functions using a Shared Runtime
- Excel Custom Functions using a JavaScript-only Runtime